docs_2.0/manual_EN/4.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -70,12 +85,80 @@ Search for vertices with properties matching a certain pattern:
     Got 2 rows (time spent 2407/3244 us)
     ```
 
-## MATCH on edge types
+* Example 3: Search for the player whose name is "Tim", return the player's relationships (edges) with others, as well as the names of these people. The `r` is a user-defined variable that represents an edge. `type(e)` to refer to the edge type.
+
+    ```nGQL
+    nebula> MATCH (v1:player{name: "Tim"}) -[e]-> (v2) \
+         -> RETURN type(r) AS Relationship, v2.name AS Name;
+    +--------------+-------------------+
+    | Relationship | Name              |
+    +--------------+-------------------+
+    | follow       | Tony Parker       |
+    +--------------+-------------------+
+    | follow       | LaMarcus Aldridge |
+    +--------------+-------------------+
+    | serve        | Warriors          |
+    +--------------+-------------------+
+    Got 3 rows (time spent 4844/6660 us)
+    ```
+
+* Example 3: Search for the team that "Tim" serves, return the team name, the start year, and the end year.
+
+    ```nGQL
+    nebula> MATCH (v1:player{name: "Tim"}) -[e:serve]-> (v2) \
+         -> RETURN v2.name AS Team_Name, e.start_year AS Start_Year, e.end_year AS End_Year;
+    +-----------+------------+----------+
+    | Team_Name | Start_Year | End_Year |
+    +-----------+------------+----------+
+    | Warriors  | 1997       | 2016     |
+    +-----------+------------+----------+
+    Got 1 rows (time spent 2768/3412 us)
+    ```
+
+* Example 4: Search for the player whose name is "Tim", find the people "Tim" follows with age above 35 and the teams they serve, and return the names of the people and teams.
```

Comment:
find the persons that the "Tim" follows and are older than 35, and find the teams that these persons serve

Reply:
The suggestion is very good. Thank you! 
However, the sentence states the "search for 'Tim'" process on purpose because the match statement does this first and then expand the search from "Tim" to others.